CRM/Twingle/Profile.php

```diff
@@ -64,6 +72,18 @@ public function logAccess() {
         WHERE name = %1", [1 => [$this->name, 'String']]);
   }
 
+  /**
+   * Copy this profile by returning a clone with all unique information removed.
+   *
+   * @return CRM_Twingle_Profile
+   */
+  public function copy() {
+    $copy = clone $this;
+    $copy->id = NULL;
+    $copy->data['selector'] = NULL;
```

Comment:
I'd vote for the `selector` property being kept when copying profiles, as it's technically just another piece of `$data`.

Reply:
But a selector should only be used in one profile, right?